src/App.js

```diff
@@ -1,9 +1,39 @@
+import Button from './components/Button';
+
 function App() {
   return (
     <div className="App">
       <header className="App-header">Hello!</header>
+      <Button
+        children="Play"
+        onClick={clickPlay}
+        icon="play-large.svg"
+        iconOnly={true}
+        size="large"
+      />
+      <Button
+        children="Play2"
+        iconOnly={true}
+        icon="play-medium.svg"
+        onClick={clickPlay2}
+      />
+      <Button
+        children="Play2"
+        iconOnly={true}
+        icon="play-medium.svg"
+        onClick={clickPlay2}
+        size="small"
+      />
```

Comment:
While it's okay for testing purposes, you shouldn't keep the "usage examples" in your commited code :)